rotors_hil_interface/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package>
```

Comment:
Currently, all other RotorS packages use the older format, so I wanted to be consistent with that. We should probably change all the packages to use the new format at some point.